Mail2Bug/Email/EWS/EWSConnectionManger.cs

```diff
@@ -58,7 +58,7 @@ public EWSConnection GetConnection(Credentials credentials)
                 }
 
                 Logger.InfoFormat("Creating FolderMailboxManager for {0}", key);
-                _cachedConnections[key] = ConnectToEWS(credentials);
+                _cachedConnections[key] = ConnectToEWS(credentials, useConversationGuidOnly);
```

Comment:
Having 'useConversationGuidOnly' as part of the actual client is not desirable. The same connection should be reusable regardless of whether we're using Guid only or the full conversation index, since that distinction is in the "Business logic" rather than the actual client or EWS interactions.